Lets Do This/Networking/DSOUserManager.m

```diff
@@ -132,6 +132,10 @@ - (void)endSessionWithCompletionHandler:(void (^)(void))completionHandler errorH
             completionHandler();
         }
     } errorHandler:^(NSError *error) {
+        [SSKeychain deletePasswordForService:[[DSOAPI sharedInstance] northstarBaseURL] account:@"Session"];
+        [SSKeychain deletePasswordForService:[[DSOAPI sharedInstance] northstarBaseURL] account:@"UserID"];
+        [self deleteAvatar];
+        self.user = nil;
```

Comment:
Same here, should be something like `deleteCurrentUser` to be called in both completion/error handlers.